src/ClimaCoupler.jl

```diff
@@ -5,6 +5,18 @@ Coupling module sufficient for initial atmos-ocean-land coupled simulation.
 """
 module ClimaCoupler
 
+using ClimaCore
+import ClimaCore: Fields, Operators
+import Logging
+
+# Disable errors from broadcasting with mismatched spaces
+Fields.allow_mismatched_diagonalized_spaces() = true
+Operators.allow_mismatched_fd_spaces() = true
+
+function __init__()
+    Logging.disable_logging(Logging.Warn) # disable warnings (for broadcasting with mismatched spaces)
+end
```

Comment:
@charleskawczynski @simonbyrne is there a better way to prevent these warnings from printing? 

One option is to remove the warning all together if we think that the error should only be disabled when coupling anyways.

Another is perhaps to change the `@warn` to `@debug` so that they are at the lowest logging level and hidden by default.

Any suggestions?